docs/getting-started/summaryofchanges.md

```diff
@@ -41,8 +41,28 @@ The following features and enhancements were added.
 
 #### API Mediation Layer
 
+#### ZSS
+
+- Added zssServer.sh, configure.sh, and start.sh in packaging [#177](https://github.com/zowe/zss/pull/177)
+- Read JWT token information from environment variables if they exist in order to further support SSO during a standard install [#178](https://github.com/zowe/zss/pull/178)
+- In previous versions, ZIS did not use the version information provided in zss/version.txt. In this version, the ZIS build uses version.txt the same way ZSS uses it [#184](https://github.com/zowe/zss/pull/184)
+- A new control flag has been added. This flag allows users to enable the LPA dev mode without recompiling the module. In the LPA dev mode, the ZWESIS01 module will always be refreshed at start up and removed from LPA upon termination. Since removing modules from LPA is considered dangerous, a new warning message will be issued to the SYSLOG and SYSPRINT if the new mode is enabled [#140](https://github.com/zowe/zowe-common-c/pull/140)
```

Comment:
I don't think this should be in the release notes. It's for developers only.

Reply:
- A new control flag has been added. This flag allows users to enable the LPA dev mode without recompiling the module. In the LPA dev mode, the ZWESIS01 module will always be refreshed at start up and removed from LPA upon termination. Since removing modules from LPA is considered dangerous, a new warning message will be issued to the SYSLOG and SYSPRINT if the new mode is enabled [#140](https://github.com/zowe/zowe-common-c/pull/140)